.github/workflows/deployNewHelp.yml

```
@@ -0,0 +1,73 @@
+name: Deploy New Help Site
+
+on:
+  # Run on any push to main that has changes to the help directory
+# TEST: Verify Cloudflare picks this up even if not run when merged to main
+#  push:
+#    branches:
+#      - main
+#    paths:
+#      - 'help/**'
+
+  # Run on any pull request (except PRs against staging or production) that has changes to the help directory
+  pull_request:
+    types: [opened, synchronize]
+    branches-ignore: [staging, production]
+    paths:
+      - 'help/**'
+
+  # Run on any manual trigger
+  workflow_dispatch:
+
+# Allow only one concurrent deployment
+concurrency:
+  group: "newhelp"
+  cancel-in-progress: false
+
+jobs:
+  build:
+    env:
+      IS_PR_FROM_FORK: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # Set up Ruby and run bundle install inside the /help directory
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
```

Whoops, I think that last commit removed the working-directory here by mistake:

```suggestion
          bundler-cache: true
          working-directory: ./help
```

---

oh whaaaat